airflow/providers/alibaba/cloud/hooks/oss.py

```diff
@@ -338,30 +338,30 @@ def get_credential(self) -> oss2.auth.Auth:
         extra_config = self.oss_conn.extra_dejson
         auth_type = extra_config.get("auth_type", None)
         if not auth_type:
-            raise Exception("No auth_type specified in extra_config. ")
+            raise ValueError("No auth_type specified in extra_config. ")
 
         if auth_type != "AK":
-            raise Exception(f"Unsupported auth_type: {auth_type}")
+            raise TypeError(f"Unsupported auth_type: {auth_type}")
```

Comment:
TypeError more related to the invalid type as object, e.g. expected sting but got list and something similar. So better to use ValueError. Maybe also there is some more better type also exists because ValueError it is also could be about a lot of things, however it way better that just Exception

```suggestion
            raise ValueError(f"Unsupported auth_type: {auth_type}")
```

Reply:
Thanks good point. Changed.